pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
 ]
-requires-python = ">=3.10"
+requires-python = ">=3.9"
```

Comment:
hey, this change needs to be reverted if you want this PR to get merged - I am sticking with `>=3.10` for the `X | Y` union syntax for type annotations.

BTW, sorry for the delay in responding, I need to do some further research to understand the websockets==14.0 changes.

Reply:
Can I replace` | `with typing.Union ? I need 3.9 compatibility …

Reply:
Sorry, I recreated MR, (can't find way to change branch in already created MR), I can't rollback python 3.9 adaptation in main and created MR with other branch : https://github.com/stephanlensky/zendriver/pull/22